New package: BcubeGmsh v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: BcubeGmsh
- Repository: https://github.com/bcube-project/BcubeGmsh.jl
- Created by: @bmxam
- Version: v0.1.0
- Commit: 9c8e5342a0a0107f4cead786983433380d50236f
- Reviewed by: @bmxam
- Reference: https://github.com/bcube-project/BcubeGmsh.jl/issues/2#issuecomment-2514957047
- Description: Implementation of Bcube IO interface for Gmsh format
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598fc97ea64007ecf7024caddf42b1c769e2bed76889cbd54e7db8d1e274f66734bc37b5e2ef445e0e63a5b3ab41ece163962d8ae4de250f8aa7c59669a003a1c3ca995ae8bdbd5af144e147bb72159eb0aeef2d9462393f71c7a4d5f172dd5049b5fe0558c2203b52892df09cf318924cae40cece9de975170831bffdb396cf80c -->

goerz commented:
This has the same issue with missing documentation as I mention in https://github.com/JuliaRegistries/General/pull/120345#issuecomment-2507974860

JuliaRegistrator committed:
New package: BcubeGmsh v0.1.0

UUID: 8c8b3700-392b-4630-b974-b8b78831232a
Repo: https://github.com/bcube-project/BcubeGmsh.jl.git
Tree: da066a7a1f0177a040dccbba98deec3f58492d73

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
